Microsoft.NET.Build.Containers/CreateNewImageToolTask.cs

```diff
@@ -176,12 +155,22 @@ protected override ProcessStartInfo GetProcessStartInfo
         string responseFileSwitch
     )
     {
+        VSHostObject hostObj = new VSHostObject(HostObject as System.Collections.Generic.IEnumerable<ITaskItem>);
+        if (hostObj.ExtractCredentials(out string user, out string pass))
+        {
+            extractionInfo = (true, user, pass);
+        }
+        else
+        {
+            Log.LogWarning("Host object failed to extract");
```

Comment:
What keeps this from firing in the "there was no host object specified" case?

Reply:
Good point. Logging a warning on false is a bit extreme for this. Turning it into a lower pri message.